scripts/local/e2e_test.sh

```diff
@@ -15,8 +15,6 @@ function printHelp() {
     printUsage
 }
 
-valid_components="teleporter governance validator-manager"
-
```

Comment:
We still reference `valid_components` on line 21. We should probably just `ls ./tests/local/` to give a list of valid components.

Reply:
I just removed the variable containing valid components altogether to simplify things.